### DISCUSSION TITLE
Anyone having issues with mux-video-react poster frames not displaying on iOS?

### DISCUSSION THREAD

**Discussion by zachkaigler** [muxinc/elements#357] [Q&A]
Not sure if I should file this as an issue because I can't confirm yet if this is happening to anyone else outside of our environment, but poster frames in the `MuxVideo` component no longer seem to be properly displaying on iOS devices as of this morning - instead of the thumbnail I'm just seeing a blank space. We have a prod deploy up that was working fine yesterday and is no longer working today with no code changes. Is anyone else experiencing this? iOS devices only - Chrome, android, and OSX Safari all seem to be working fine.

**Comment by zachkaigler** [comment]
Decided to open an issue for this [here](https://github.com/muxinc/elements/issues/358).

**Comment by luwes** [comment]
Thanks for the feedback. I quickly took a look at this and I don't believe anything changed in `MuxVideo`.

`MuxVideo` still requires the user to set the `poster` property manually to get a thumbnail when the video is not playing yet.

It's possible that because iOS is in energy saving mode or using cellular data it will not preload any video content (preload=none)
and as a result don't show any loaded video content frame.

Probably the easiest solution would be to add a poster prop with the Mux thumbnail endpoint.
See https://docs.mux.com/guides/video/get-images-from-a-video#get-an-image-from-a-video

Hope this helps!

**Reply by zachkaigler** [reply]
That's definitely odd - we had no problems with the thumbnails displaying automatically without a manual `poster` prop before yesterday, and this is happening across all iPhones that we've checked on, several of which that were on wifi and not in power saver mode. Dylan also reported seeing the same thing on his iPhone with the codepen I mentioned in the bug report during a twitter conversation I had with him [here](https://twitter.com/dylanjha/status/1565097095153221632). I'll see if I can find a quick solution within our architecture to pipe the thumbnail into the `poster` prop, but I'm still a bit puzzled as to why this only started happening out of the blue yesterday.
